contracts/farms/common/RevoFarmBot.sol

```diff
@@ -434,6 +287,14 @@ contract UbeswapFarmBot is ERC20, AccessControl, Pausable {
         );
```

Comment:
this "Compound" event either needs to get taken out or renamed, I think.

Reply:
I think moving all the events/constants to a generic `RevoEvents` library seems like a good idea.

Reply:
Errr... [bit of a problem](https://ethereum.stackexchange.com/questions/11137/watching-events-defined-in-libraries) with this approach. Not actually sure if this is worth doing if we just have to redefine the events in the contracts that use them.

Reply:
I've left this as-is for now; I'm not sure what the best solution here is given this weird constraint.